assets/component-mega-menu.css

```diff
@@ -0,0 +1,38 @@
+.mega-menu {
+  position: initial;
+}
+
+.mega-menu-content {
+  background-color: rgb(var(--color-background));
+  border-left: 0;
+  border-radius: 0;
```

Comment:
Just a thought/question. All of our other popups and dropdowns preserve the radius for the bottom left and right. Do we want to offer this for the mega nav? On mobile, even though it's full width, we do include the border bottoms on the cart notification. But if we do, we may need to add the borders right and left as well.

<img width="1197" alt="Screen Shot 2022-03-04 at 4 31 33 PM" src="https://user-images.githubusercontent.com/26778637/156844415-bc985bbc-bcd5-43ee-949d-1587727baf22.png">

<img width="396" alt="Screen Shot 2022-03-04 at 4 33 08 PM" src="https://user-images.githubusercontent.com/26778637/156844585-4fa9e101-7a2b-4bb8-8c73-575c7d335540.png">

<img width="1193" alt="Screen Shot 2022-03-04 at 4 44 24 PM" src="https://user-images.githubusercontent.com/26778637/156845938-187df2ec-233f-43d0-af09-cf3b9f724d1f.png">